forward_engineering/helpers/alterScriptHelpers/alterEntityHelper.js

```diff
@@ -1,13 +1,35 @@
+/**
+ * @typedef {Object} Collection
```

Comment:
Better to extract complex types into own js files into "types" folder - see postgres for reference